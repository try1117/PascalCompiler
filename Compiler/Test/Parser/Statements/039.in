program test;
const
  pi: double = 3.14159265358979;
var
  pi2: double = 2 * pi;
  pi3: double = pi + pi + pi;
  pi3_: double = pi + pi2;
begin
  write('pi + 1 = ', pi + 1, '2 * pi = ', pi2);
end.
