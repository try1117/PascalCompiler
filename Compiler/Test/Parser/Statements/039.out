Lexical exception in position (7,21) - Illegal expression
